### PR TITLE
[tflite] fix a dilated dw conv problem in tflite_convert

### PR DESCRIPTION
**Pull request by freedomtan**
In tflite_convert/toco, it's assumed that there is a BiasAdd
before or after a BatchToSpaceND. However, in some frozen
graph_def files, e.g., DeepLab V3 mobilenet v2 coco
checkpoints [1], it's a fused BatchNormalization rather than
a BiasAdd.

This patch adds the BatchNorm case.

[1] https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md

### CODE REVIEW

**Comment by gbaned** [comment]
@daverim Can you please take a look at this PR? Thanks!

**Comment by freedomtan** [comment]
this fixed https://github.com/tensorflow/tensorflow/issues/29509 in the original non-MILR `tflite_covert`

**Comment by jdduke** [comment]
We're no longer pushing updates to the TOCO path, any chance this needs to be added to the MLIR-based conversion path?

**Comment by freedomtan** [comment]
@jdduke The MLIR-based conversion could handle this case already. So, I'll close this.
